docs/guided-tour/getting-started.md

```diff
@@ -30,14 +31,14 @@ To complete this tutorial, you will need:
 - Basic knowledge of Git
 - Basic knowledge of the command line
 - A [Semaphore account][semaphore-home]
-- A [GitHub account][github]
+- A [GitHub][github] or [Bitbucket](http://bitbucket.com/) account
 
 ## Creating a Git repository
 
-Start by creating a new Git repository on GitHub. You may skip this section if you
+Start by creating a new Git repository on GitHub or Bitbucket. You may skip this section if you
 plan to use an existing repository.
 
-- Navigate to GitHub and [create a new repository][github-new].
+- Create a new repository on [Github][github-new] or [Bitbucket](https://bitbucket.org/new/)
```

Comment:
follow the pattern of formatting links that's used throughout the page, where the name/URL pairs are defined at the bottom